arch/arm64/boot/dts/qcom/msm8953-huawei-milan.dts

```diff
@@ -0,0 +1,549 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright (c) 2023, bzy-080408<Bzy080408@outlook.com>
+ * Copyright (c) 2023, Clrs17<2318678921@qq.com>
+ */
+/dts-v1/;
+
+#include "msm8953.dtsi"
+#include "pm8953.dtsi"
+#include "pmi8950.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/interrupt-controller/irq.h>
+#include <dt-bindings/leds/common.h>
+
+
+/ {
+	model = "Huawei Maimang 5 / Nova (Plus) / G9 (Plus)";
+	compatible = "huawei,milan", "qcom,msm8953", "lk2nd,device";
+	qcom,msm-id = <293 0x0>;
+	qcom,board-id = <8000 0>, <8001 0>, <8002 0>, <8003 0>, <8016 0>,
+			<8017 0>, <8018 0>, <8019 0>, <8032 0>, <8033 0>,
+			<8035 0>, <8048 0>, <8049 0>, <8050 0>, <8051 0>,
+			<8066 0>, <8064 0>, <8067 0>, <8080 0>, <8082 0>,
+			<8083 0>, <8098 0>, <8099 0>, <8114 0>, <8130 0>;
+	chassis-type = "handset";
+	
+	battery: battery {
+		compatible = "simple-battery";
+		charge-full-design-microamp-hours = <3270000>;
+		constant-charge-current-max-microamp = <2000000>;
+		voltage-min-design-microvolt = <3400000>;
+		voltage-max-design-microvolt = <4400000>;
+	};
+	
+	vph_pwr: vph-pwr-regulator {
```

Comment:
please move down to be in alphabetic order

Reply:
fixed